tests/Aspire.EndToEnd.Tests/IntegrationServicesFixture.cs

```diff
@@ -22,7 +22,7 @@ public sealed class IntegrationServicesFixture : IAsyncLifetime
 
     public Dictionary<string, ProjectInfo> Projects => _projects!;
 
-    public ProjectInfo IntegrationServiceA => Projects["integrationservicea"];
+    public ProjectInfo   => Projects["integrationservicea"];
```

Comment:
This looks broken

Reply:
Thanks. Must have fat fingered it as I was jumping around the IDE.